src/components/metadata/metadata.js

```diff
@@ -89,8 +89,12 @@ const MetaData = ({
     return string?.replace(/^\//g, '');
   };
 
-  const shortenDatasetType = (string) => {
-    return string?.split('.').pop();
+  const shortenDatasetType = (value) => {
+    const isList = Array.isArray(value);
+
+    return isList
+      ? value.map((val) => val.split('.').pop())
+      : value?.split('.').pop();
```

Comment:
Maybe just this? If you want to set a variable, I'd use `isArray` for the name. `list` is from Python.

```suggestion
    return Array.isArray(value)
      ? value.map((val) => val.split('.').pop())
      : value?.split('.').pop();
```